### ISSUE TITLE
Modal window does not scroll, breaks manager layout, when too tall for the viewport

### ISSUE THREAD

**Issue by dev-willis** [modxcms/revolution#16285]
### Summary
When a modal window is opened which contains more content than will fit in the window it breaks the manager layout.

### Step to reproduce
Right-click a template in use by many resources, select "Delete Template" and click "Yes."

### Observed behavior
If the modal window is taller than the viewport it will be aligned with the bottom of viewport and the top will be inaccessible. After clicking "Okay," only a sliver of the manager is visible at the top of the viewport. Refreshing the page restores the layout. Resizing the window also restores it, although only partially in some cases.
![image](https://user-images.githubusercontent.com/89114332/196043441-99a3455f-233c-4733-8ece-4beb1dd43f49.png)
![image](https://user-images.githubusercontent.com/89114332/196043447-5410e394-030e-4cab-8acb-f02e7052f7ad.png)

### Expected behavior
Perhaps the modal window should scroll?

### Environment
MODX 3.0.1, MODX Cloud

**Comment by smg6511** [comment]
What is generating the modal? Is it an Extra or custom component?

**Comment by dev-willis** [comment]
Neither. It's part of MODX core.

**Comment by smg6511** [comment]
Ok. Just wanting to understand what would result in such a long modal ... what triggers the modal you show in the screenshot?

**Comment by dev-willis** [comment]
Right-click a template in use by many resources, select "Delete Template" and click "Yes."

**Comment by smg6511** [comment]
Ok, great. That will help in solving the issue. I'll take a closer look and see what we can do.
